cmake/compile_definitions/linux.cmake

```diff
@@ -155,14 +155,20 @@ endif()
 
 # tray icon
 if(${SUNSHINE_ENABLE_TRAY})
+    set(TRAY_VERSION 0)
     pkg_check_modules(APPINDICATOR appindicator3-0.1)
     if(NOT APPINDICATOR_FOUND)
+        set(TRAY_VERSION 2)
         pkg_check_modules(APPINDICATOR ayatana-appindicator3-0.1)
+    else()
+        set(TRAY_VERSION 1)
```

Comment:
@TheElixZammuto can we use this property to determine which imports to you in the tray library?

```txt
#19 113.4 /build/sunshine/third-party/tray/tray_linux.c:2:10: fatal error: libayatana-appindicator/app-indicator.h: No such file or directory
#19 113.4     2 | #include <libayatana-appindicator/app-indicator.h>
#19 113.4       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#19 113.4 compilation terminated.
```

The new library is not available on Fedora as far as I know.